packages/client-api-schema/src/notifications.ts

```diff
@@ -4,9 +4,11 @@ import {ChannelResult, Message, DomainBudget} from './data-types';
 // these notifications come *from* the wallet, which is not strictly how JSON-RPC should work
 // (since we treat the wallet as the 'server')
 
-export type ChannelProposedNotification = JsonRpcNotification<'ChannelProposed', ChannelResult>;
 export type ChannelUpdatedNotification = JsonRpcNotification<'ChannelUpdated', ChannelResult>;
+// TODO: Deprecate these. They are redundant, given the `ChannelResult`.
```

Comment:
They're still relevant for the `xstate-wallet` which uses the JsonRpc interface

Reply:
That's fine. They can use the `ChannelUpdatedNotification` with the `'proposed'`, `'closed'` or `'closing'` status.